### PR TITLE
Adjust the azure model deployment name call corresponding to README

### PR DESCRIPTION
**Pull request by zjy282**
According to the modification of #306, adjust the azure model deployment name call corresponding to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -436,7 +436,13 @@ import (
 
 func main() {
 
-	config := openai.DefaultAzureConfig("your Azure OpenAI Key", "https://your Azure OpenAI Endpoint ", "your Model deployment name")
+	config := openai.DefaultAzureConfig("your Azure OpenAI Key", "https://your Azure OpenAI Endpoint")
+	config.AzureModelMapperFunc = func(model string) string {
+        azureModelMapping = []string{
```
Comment (sashabaranov) [inline README.md:441]: Thank you for the PR! Could you please comment out the `config.AzureModelMapperFunc` section and clearly indicate that it is optional?

Comment (zjy282) [reply README.md:441]: Of course, great idea

**Comment by sonicrang** [comment]
I've try it in my application and it should be:

```
config.AzureModelMapperFunc = func(model string) string {
			azureModelMapping := map[string]string{
				defaultModelType: deploymentName,
			}
			return azureModelMapping[model]
		}
```

**Comment by zjy282** [comment]
> 



> 我已经在我的应用程序中尝试过它，它应该是：
> 
> ```
> config.AzureModelMapperFunc = func(model string) string {
> 			azureModelMapping := map[string]string{
> 				defaultModelType: deploymentName,
> 			}
> 			return azureModelMapping[model]
> 		}
> ```

This problem has been fixed, thanks

**Review by sashabaranov — APPROVED** [review]

**Pull request merged**